Competition2017/src/competition/subsystems/feeder/FeederSubsystem.java

```diff
@@ -0,0 +1,59 @@
+package competition.subsystems.feeder;
+
+import org.apache.log4j.Logger;
+import com.ctre.CANTalon.TalonControlMode;
+import com.google.inject.Inject;
+
+import competition.subsystems.feeder.SideFeederSubsystem.FeederSide;
+import xbot.common.command.BaseSubsystem;
+import xbot.common.controls.actuators.XCANTalon;
+import xbot.common.injection.wpi_factories.WPIFactory;
+import xbot.common.properties.DoubleProperty;
+import xbot.common.properties.XPropertyManager;
+
+public class FeederSubsystem extends BaseSubsystem{
+
+        private static Logger log = Logger.getLogger(FeederSubsystem.class);
+
+        protected final DoubleProperty intakePowerProperty;
+        protected final XCANTalon feederMotor;
+        
+        private SideFeederSubsystem leftFeeder;
+        private SideFeederSubsystem rightFeeder;
+
+        @Inject
+        public FeederSubsystem(WPIFactory factory, XPropertyManager propManager){
+
+            log.info("Creating Feeder Subsystem");
+
+            feederMotor = factory.getCANTalonSpeedController(5);
+            feederMotor.setBrakeEnableDuringNeutral(false);
+            feederMotor.setProfile(0);
+            feederMotor.setControlMode(TalonControlMode.PercentVbus);
+
+            intakePowerProperty = propManager.createPersistentProperty("Feeder intake power", 0.5);
+
+            SideFeederSubsystem leftFeeder = new SideFeederSubsystem(31, FeederSide.Left, factory, propManager);
+            SideFeederSubsystem rightFeeder = new SideFeederSubsystem(44, FeederSide.Right, factory, propManager);
```

Comment:
I'll add an item to our backlog to update all motors with their proper values from the electrical contract (once that has been finalized)